iam_access_analyzer/README.md

```diff
@@ -33,7 +46,8 @@ This integration does not send events
 
 ## Troubleshooting
 
-Need help? Contact [Datadog support][7].
+Need help? Contact [Datadog support][3].
 
-[1]: https://docs.datadoghq.com/integrations/iam_access_analyzer/
-[7]: https://docs.datadoghq.com/help
+[1]: /integrations/amazon_web_services/?tab=allpermissions#set-up-the-datadog-lambda-function
```

Comment:
```suggestion
[1]: /logs/guide/forwarder/
```

Reply:
@anshumgargdd Can you confirm if these are the right links?

Reply:
Thanks for the suggestions @buraizu 
In the spirit of getting this migration landed, I committed your suggestions. 

@anshumgargdd If any of these don't seem right, I'll defer to you for followup changes 🙂